package.json

```diff
@@ -74,10 +79,13 @@
 		"preset": "ts-jest"
 	},
 	"type": "module",
-	"svelte": "./index.js",
 	"exports": {
 		".": {
-			"svelte": "./index.js"
+			"types": "./dist/index.d.ts",
+			"svelte": "./dist/index.js"
```

Comment:
gotta export types and svelte components

https://kit.svelte.dev/docs/packaging#anatomy-of-a-package-json-exports